core/src/main/java/hudson/model/StringParameterDefinition.java

```diff
@@ -69,7 +70,7 @@ public StringParameterValue getDefaultParameterValue() {
         return v;
     }
 
-    @Extension
+    @Extension @Symbol({"string","stringParam"})
```

Comment:
> So far `@Symbol` was paired up with `@Extension` (which means it's on `Descriptor` and not on `Describable`).

Hmm, this clashes with my original understanding of the design and the way that `DescribableModel` works, which is at the `Describable` level. Does need more thought.